src/R/Support/Impl/Help/Packages/PackageInfo.cs

```diff
@@ -2,257 +2,113 @@
         /// </summary>
-        public IReadOnlyCollection<INamedItemInfo> Functions {
-            get {
-                IReadOnlyCollection<INamedItemInfo> functions = _functionIndex.GetPackageFunctions(this.Name);
-                if (functions == null || functions.Count == 0) {
-                    functions = LoadFunctionInfoFromPackageHelpIndex();
-                }
-
-                return functions;
-            }
-        }
-        #endregion
-
-        /// <summary>
-        /// Extract package description from the DESCRITION file on disk
-        /// </summary>
-        private string GetDescription() {
-            StringBuilder sb = new StringBuilder();
-            bool found = false;
-
-            try {
-                // DESCRIPTION uses a simple file format called DCF, the Debian control format. 
-                // Each line consists of a field name and a value, separated by a colon.
-                // When values span multiple lines, they need to be indented:
-                //
-                // Description: The description of a package is usually long,
-                //    spanning multiple lines. The second and subsequent lines
-                //    should be indented, usually with four spaces.
-
-                string descriptionFilePath = Path.Combine(this.InstallPath, this.Name, "DESCRIPTION");
-
-                using (StreamReader sr = new StreamReader(descriptionFilePath, Encoding.UTF8)) {
-                    while (!found) {
-                        string line = sr.ReadLine();
-                        if (line == null) {
-                            break;
-                        }
-
-                        if (line.StartsWithIgnoreCase("Description:")) {
-                            line = line.Substring(12).Trim();
-                            sb.Append(line.Trim());
-                            sb.Append(' ');
-
-                            while (!found) {
-                                line = sr.ReadLine();
-                                if (line == null) {
-                                    break;
-                                }
-
-                                if (line.Length > 0 && char.IsWhiteSpace(line[0])) {
-                                    line.Trim();
-
-                                    sb.Append(line.Trim());
-                                    sb.Append(' ');
-                                } else {
-                                    found = true;
-                                    break;
-                                }
-                            }
-                        }
+        public IEnumerable<INamedItemInfo> Functions => _functions;
+
+        public void WriteToDisk() {
+            if (!_saved) {
+                var filePath = GetCacheFilePath();
+                try {
+                    var dir = Path.GetDirectoryName(filePath);
+                    if (!Directory.Exists(dir)) {
+                        Directory.CreateDirectory(dir);
```

Comment:
I meant to verify that we did actually write file to disk for example. I used MS in drag-drop tests but that's b/c DD already gets stream from clipboard.